src/badguy/dispenser.cpp

```diff
@@ -163,6 +163,12 @@ Dispenser::active_update(float dt_sec)
   }
 }
 
+void
+Dispenser::kill_fall()
+{
+  return;
```

Comment:
Isn't the return kinda pointless? You can just leave the method empty.

Reply:
No. It overrides the kill_fall method to do absolutely nothing, making the dispenser truly unkillable

Reply:
wait now i get what you mean. Sure i can remove the return